### PR TITLE
Formik for QuestionsForm.js has been created

### PR DESCRIPTION
**Pull request by whitneyxlachelle**
I changed some of the styling so that it can be more cohesive with the marketing page.

### CODE REVIEW

**Review by Jason-Aviles — APPROVED** [review]

**Review by hutchcrowley — APPROVED** [review]
Can you add your node_modules folder and package files to your gitignore and submit a new pull request? We're having a merge conflict.

**Review by hutchcrowley — CHANGES_REQUESTED** [review]
Please see merge conflict. I think you might need to add your package files to your gitignore file. Ask Jason for more info.

File: src/styles/answersStyles.js [hunk 1]
```diff
@@ -23,7 +24,18 @@ border: none;
 
 //Styles for Answer.js
 
-export const AnswerSection = styled.div `
+export const AnswerContainer = styled.div`
```
Comment (hutchcrowley) [inline src/styles/answersStyles.js]: I styled our Answer, AnswerList, and some styles on QuestionList. Check them out and let me know if you want to keep mine, incorporate yours, or what. Thanks! Not sure why this wouldn't just let me merge everything anyway.

File: src/components/Answers/AnswerForm.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ function AnswerForm(props) {
 
         <AnswerWrapper >   
             <form className="answer-form" onSubmit={handleSubmit}>
-                <textarea name="answer" placeholder="Answer" value={newAnswer.answer} onChange={handleChange}  />
+                <textarea name="answer" rows="4" cols="50" placeholder="Answer" value={newAnswer.answer} onChange={handleChange}  />
```
Comment (hutchcrowley) [inline src/components/Answers/AnswerForm.js]: Hey I completely re-did AnswerForm to use Formik. check it out and let me know if you want to keep these changes or no.

File: src/components/Answers/Answer.js [hunk 3]
```diff
@@ -1,17 +1,16 @@
 import React from "react"
 
-
 // Style files
 //import answersStyles from '../../styles/answersStyles';
-import { AnswerSection } from '../../styles/answersStyles';
+import { AnswerContainer } from '../../styles/answersStyles';
```
Comment (hutchcrowley) [inline src/components/Answers/Answer.js]: Please check my updated styles for the Answer component and if you want to keep these changes, we can. Not sure why it didn't just let me pull this in..

**Review by hutchcrowley — CHANGES_REQUESTED** [review]

**Review by whitneyxlachelle — COMMENTED** [review]
I resolved the merging conflicts.

**Pull request merged**